specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/examples/ComposeModels.json

```diff
@@ -4,9 +4,9 @@
     "Content-Type": "application/json",
     "Ocp-Apim-Subscription-Key": "{API key}",
     "body": {
-         "composeRequest": {
-            "ModelsIds": ["ModelA", "ModelB"]
-        }
+    },
+    "composeRequest": {
+      "ModelsIds": ["ModelA", "ModelB"]
```

Comment:
It is helpful to use a descriptive name for the models.  But I believe ModelA and ModelB should be GUIDs here.